lldb/test/API/lang/cpp/dwim-print-fixit/TestCppDWIMPrintFixIt.py

```diff
@@ -0,0 +1,27 @@
+"""
+Tests whether the do-what-I-mean (DWIM) print `po` alias applies FixIts like `expr` does
+"""
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+
+
+class CPP_DWIM_Fixit_TestCase(TestBase):
+    def test_with_run_command(self):
+        "Confirm that the `po` command (alias) applies a FixIt " \
+        "and prints it out to the console, " \
+        "just like the `expression` command."
```

Comment:
I tried using a multiline string, but that adds hard returns and all the indentation spaces in the middle of the string, which means you get this on the output console …

```
 Confirm that the `po` command (alias) applies a FixIt
        and prints it out to the console,
        just like the `expression` command.
```

Instead of this…
```
 Confirm that the `po` command (alias) applies a FixIt and prints it out to the console, just like the `expression` command.
```

Plus, I noticed that running the single unit test only prints the string up until the first `\n` to the console, which means it's truncating everything after it.

When I use line `\` continuations, it shows the whole string without truncating it, which was the only way I could figure out how it to work without making the source line too long (in Python).

Reply:
Fixed with a `"""` string that's one line for doxygen.

```"""Confirms `po` shows an expression after applying Fix-It(s)."""```